packages/storage/src/implementation/requests.ts

```diff
@@ -218,7 +218,7 @@ export function metadataForUpload_(
   blob: FbsBlob,
   opt_metadata?: Metadata | null
 ): Metadata {
-  let metadata = object.clone<Metadata>(opt_metadata);
+  let metadata = clone(opt_metadata);
```

Comment:
This could be `Object.assign({}, opt_metadata)`.

Reply:
Done in next PR.